discord/sticker.py

```diff
@@ -414,7 +414,7 @@ class GuildSticker(Sticker):
 
     def _from_data(self, data: GuildStickerPayload) -> None:
         super()._from_data(data)
-        self.available: bool = data['available']
+        self.available: Optional[bool] = data.get('available')
```

Comment:
This is a breaking change.
```suggestion
        self.available: bool = data.get('available', True)
```